docs/stdlib/array.rst

```diff
@@ -141,14 +140,15 @@ Reference
         db> select [(x := 1, y := 1), (x := 2, y := 3.3)][1];
         {(x := 2, y := 3.3)}
 
-    Negative indexing is supported:
+    This operator may also be used when trying to negatively index elements.
```

Comment:
I don't like the "trying" word here. There's no trying - you just index :)

Reply:
Will change